src/contracts/core/RewardsCoordinatorStorage.sol

```diff
@@ -89,6 +89,9 @@ abstract contract RewardsCoordinatorStorage is IRewardsCoordinator {
     /// if rewards submission hash for all stakers and operators has been submitted
     mapping(address => mapping(bytes32 => bool)) public isRewardsSubmissionForAllEarnersHash;
 
+    /// @notice Mapping: avs => avsPerformanceRewardsSubmissionHash => bool to check if performance rewards submission hash has been submitted
+    mapping(address => mapping(bytes32 => bool)) public isAVSPerformanceRewardsSubmissionHash;
```

Comment:
Was there a reason we needed separate mappings for each reward type? I just copied the current setup but questioning the need for it here.

Reply:
let's combine into one, agreed

Reply:
I'm assuming this is used by the rewards pipeline/sidecar in some form. I don't want to break backwards compatibility if that's the case.

Reply:
@ypatil12 @8sunyuan may have opinions

Reply:
You can have collisions, in the worst case, between `AVSRewardsSubmission`, `RewardsSubmissionForAllEarners`, and `RewardsSubmissionForAll`  if they are submitted with the same params in the same block. You should be able to combine all the AVS submitted reward types into one mapping.

Reply:
This should be fine since we include the nonce in the rewards submission hash. cc @8sunyuan to work

Reply:
Internally we're not using this mapping offchain

Reply:
Oh if it's not being used offchain, what was the purpose of this mapping? I don't see it being used anywhere else besides just setting it during reward submission.

Reply:
@8sunyuan do you remember? I can't recall off the top of my head

Reply:
It appears it may actually have no use... potentially we wanted it for some sort of future pipeline integration

Reply:
Keeping it for uniformity's sake.